yarl/__init__.py

```diff
@@ -192,6 +192,45 @@ def __init__(self, val='', *, encoded=False, strict=False):
         self._val = val
         self._cache = {}
 
+    @classmethod
+    def build(cls, *, scheme='', user='', password='', host='', port=None, path='',
+              query=None, query_string='', fragment='', strict=False):
+
+        """ Creates and returns a new URL
+
+        :type scheme: str
```

Comment:
Pls remove these lines, we don't use sphinx autodoc for the project.

Docstring hould be just 
```
"""Creates and returns a new URL.
"""
```
(no leading space and period at the end).
Just look on other docstrings from the module.